docker-compose.prod.yml

```diff
@@ -2,7 +2,5 @@ version: '2'
 services:
   web:
     command: /bin/bash -c "cd src && python manage.py collectstatic --noinput && /usr/local/bin/gunicorn sous-chef.wsgi:application -w 2 -b :8000"
-    volumes:
-      - /code/src/sous-chef/static
     expose:
       - "8000"
```

Comment:
This "expose" clause comes up in the prod file and in the override file. Can it be brought back to docker-compose.yml?